README.md

```diff
@@ -25,11 +28,35 @@
 ## Technologies
 HTML, CSS, JavaScript, Local Storage
 
+## Contributing
+When contributing to this repository, please first discuss the change you wish to make via issue  with the owners of this repository before making a change.
+Please note we have a [code of conduct](docs/code-of-conduct), please follow it in all your interactions with the project.
+
+## Pull Request Process
+We love pull requests from everyone. By participating in this project, you agree to abide by the MoTo [code of conduct](docs/code-of-conduct)).
+
+1. Fork, then clone the repo:
+```
+git clone https://github.com/chingu-voyages/v13-toucans-team-03.git
+```
+
+2. Change into the MoTo directory:
+```
+cd v13-toucans-team-03
+```
+
+3. Push to your fork and submit a pull request.
+At this point, you're waiting on us. We will try to get to your pull request as soon as possible.. We may suggest some changes or improvements or alternatives.
+Some things that will increase the chance that your pull request is accepted:
+    -  Write tests.
+    - Follow CSS BEM convention.
+    - Write a good commit message.
+
```

Comment:
@sophi-li Great work! I think we might want to tell the contributor to create a new branch for the feature they are working on.